builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -332,175 +259,156 @@ node  {
+    } else if (status == 'COMPLETED') {
+        echo "Send Build Completed notification"
+        body_subject = "Jazz Build Notification: Creation COMPLETED successfully for service:  ${service_config['service']} "
+    } else {
+        echo "Unsupported build status, nothing to email.."
+        return
+    }
+    
+    body_text = "${body_text} \n\n For Service: ${service_config['service']}  in Domain:  ${service_config['domain']}"
+	
+    body = JsonOutput.toJson([
+      from : 'Jazz Admin <' + configLoader.JAZZ.STACK_ADMIN + '>',
+      to : service_config['created_by'],
+      subject : body_subject,
+      text : body_text,
+      cc : cc_email,
+      html : body_html
+    ])
 
    	try {
-		def sendMail = sh(script: "curl -X POST \
-						${g_base_url}/jazz/email \
-						-k -v -H \"Authorization: $auth_token\" \
-						-H \"Content-Type: application/json\" \
-						-d \'${body}\'", returnStdout: true).trim()
-		def responseJSON = parseJson(sendMail)
-
-		if (responseJSON.data) {
-			echo "successfully sent e-mail to ${service_config['created_by']}"
-		} else {
-			echo "exception occured while sending e-mail: $responseJSON"
-		}
+      def sendMail = sh(script: "curl -X POST \
+              ${g_base_url}/jazz/email \
+              -k -v -H \"Authorization: $auth_token\" \
+              -H \"Content-Type: application/json\" \
+              -d \'${body}\'", returnStdout: true).trim()
+      def responseJSON = jsonParse(sendMail)
+
+      if (responseJSON.data) {
+        echo "Successfully sent email to ${service_config['created_by']}"
+      } else {
+        echo "Exception occured while sending email: $responseJSON"
+      }
    	} catch (e) {
        	echo "Failed while sending build status notification"
    	}
 }
 
-@NonCPS
-def parseJson(jsonString) {
-    def lazyMap = new groovy.json.JsonSlurper().parseText(jsonString)
-    def m = [:]
-    m.putAll(lazyMap)
-    return m
-}
 
 /*
-* Load build module
+* Loading build modules
 */
-
 def loadBuildModules(buildModuleUrl){
-	echo "loading env variables, checking repos..."
-
+	echo "Loading build modules.."
 	dir('build_modules') {
-		checkout([$class: 'GitSCM', branches: [
-			[name: '*/master']
-		], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [
-				[credentialsId: repo_credential_id, url: buildModuleUrl]
-			]])
+		checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [
+			[credentialsId: repo_credential_id, url: buildModuleUrl]
+		]])
 
 		configModule = load "config-loader.groovy"
 		configLoader = configModule.loadConfigData(aws_credential_id, region, instance_prefix)
 		echo "config loader loaded successfully."
 
 		awsAPIGatewayModule = load "aws-apigateway-module.groovy"
-    	awsAPIGatewayModule.initialize()
+    awsAPIGatewayModule.initialize()
```

Comment:
This formatting isn't shown as screwed up when I open the file. Feel free to fix it if you can.

Reply:
fixed formatting